requirements.txt

```diff
@@ -16,4 +16,5 @@ pandas==1.1.3
 psutil==5.7.2
 Pillow==8.0.0
```

Comment:
I think you mentioned needing to upgrade mne for this - what version are you on?

Reply:
I did some testing and the issue I was encountering wasn't related to mne. I left the version as-is for this review so we could do more stress-testing on it.